build-image/first-boot.bash

```diff
@@ -199,26 +187,25 @@ else
 fi
 echo "$(timestamp) [openHABian] Execution of 'openhabian-setup.sh unattended' completed."
 
-echo -n "$(timestamp) [openHABian] Waiting for openHAB to become ready on $HOSTNAME ..."
+echo -n "$(timestamp) [openHABian] Waiting for openHAB to become ready on ${HOSTNAME}... "
 
 # this took ~130 seconds on a RPi2
-if tryUntil "wget -S --spider -t 3 --waitretry=4 http://${HOSTNAME}:8080/start/index 2>&1 | grep -q 'HTTP/1.1 200 OK'" 20 10; then echo "FAILED"; exit 1; fi
-echo "OK"
+if ! tryUntil "wget -S --spider -t 3 --waitretry=4 http://${HOSTNAME}:8080/start/index |& grep -q 'HTTP/1.1 200 OK'" 20 10; then echo "OK"; else echo "FAILED"; exit 1; fi
 
 echo "$(timestamp) [openHABian] Visit the openHAB dashboard now: http://${HOSTNAME:-openhab}:8080"
 echo "$(timestamp) [openHABian] To gain access to a console, simply reconnect."
 echo "$(timestamp) [openHABian] First time setup successfully finished."
 sleep 12
-if hash python3 2>/dev/null; then bash /boot/webif.bash inst_done; fi
+if [[ -x $(command -v python3) ]]; then bash /boot/webif.bash inst_done; fi
 sleep 12
 
-if [ -z "$SILENT" ]; then
+if running_in_docker; then
```

Comment:
why change behaviour here?

Reply:
Cleaning up install output, `cond_echo` assumes that it should output because Silent is not set so I had to set silent to fix it and as such used a different behavior.